Makefile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 test:
-	python3 -m unittest discover
+	python -m unittest discover
```

Comment:
I expect the code I wrote only works on Python 3, and [PEP 394][1] suggests `python3` for that.

[1]: https://www.python.org/dev/peps/pep-0394/#recommendation